### ISSUE TITLE
Add a set of achievements to encourage diverse gameplay

### ISSUE THREAD

**Issue by Xynth22** [FreeAllegiance/Allegiance-AZ#8]
Create a set of non-grind achievements that encourage new players to try out all aspects of the game.

Current list:

These Should be straightforward and will be my first batch:
First Miner Kill
First Con Kill
First Force to Eject
First Base Kill
First Base Cap

Scout set:
First Prox Kill
First probe deploy in enemies home sector
Deploy 20 probes
Spot 5 alephs
Heal 100,000 Damage with nan

**Comment by Xynth22** [comment]
I would appreciate any creative assistance on either the icons (both an active and inactive) and naming the achievements with more interesting things than "First Miner Kill".

**Comment by LordBlacksun** [comment]
Can we set achievements  based on cumulative damage repaired by scouts?

EG: "achievement granted for healing 100,000 damage"

**Comment by BackTrak** [comment]
Hey Xynth, I would like to use as many cheesy 80's graphics as we can find for this. For the beta achievement, I used a shot of Beta from the last star fighter working on his head. The good news is you can use place holders for these now (gray trophy, gold trophy), and we can update them later. 

First Miner Kill - Blood in the water - You got your first miner kill. Now go get the rest!
First Con Kill - The one that didn't get away - You got your first constructor kill! Team work makes it happen!
First Force to Eject - Tiny House - First time in a pod. It's OK, it happens to all of us. 
First Base Kill - If they build it - Your first base kill. This achievement was a lot easier than trying to get that mounted.
First Base Cap - This is my land - Base Captured! You just earned the right to yell it upon entering any drinking establishment, world wide. 

Scout set:
First Prox Kill - Can't get enough of those prox dropping beats - With just a little bit of timing, and a whole lot of enemy speed, you bagged your first prox mine kill.
First probe deploy in enemies home sector - Peeping Tom - You put down a probe in the enemies sector. 
Deploy 20 probes - Privacy Advocate - No one likes to be watched. You shut down 20 enemy probes!.
Spot 5 alephs - Early Bird - You got out there any found 5 alephs. Nice hunting!

**Comment by BackTrak** [comment]
@LordBlacksun yes, that is possible. Steam achievements can be setup as "progressive", so you can earn points towards them. @Xynth22 if you do that, you don't commit the stats right away. Those would be saved at the end of the game when stats are committed for all players on the game over screen.

**Comment by Xynth22** [comment]
Thanks BT, but I'm sorry I was hoping to spread the work not pile more back on to you :)

**Comment by BackTrak** [comment]
I can't help it, achievements get me excited!! 🥇

**Comment by Xynth22** [comment]
I have images and descriptions for everything. I still need to remake the B/W version though. Yeah I was planning to save stats at the end and only for 5v5 or more. I was going to try to find a way to require 5v5 for achievements as well.
